packages/terra-core-docs/src/terra-dev-site/doc/hyperlink/example/HyperlinkButton/ExternalHyperlinkButton.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import HyperlinkButton from 'terra-hyperlink/lib/HyperlinkButton';
+
+export default () => <HyperlinkButton text="External hyperlink" onClick={() => {}} variant="external" />;
```

Comment:
The above examples include the word "button" in the next whereas this and other examples do not. Not a big deal but might be good to stay consistent.

Reply:
Done [f6ad624](https://github.com/cerner/terra-core/pull/3608/commits/f6ad624f5c223379ebc4c01bca2cad68e7697e96)